ui/src/components/content/catalog/components/ServiceProvider.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { Tabs } from 'antd';
+import ServiceDetail from './ServiceDetail';
+import { useEmotionCss } from '@ant-design/use-emotion-css';
+import { CategoryOclVo, OclDetailVo, ProviderOclVo, VersionOclVo } from '../../../../xpanse-api/generated';
+import { Tab } from 'rc-tabs/lib/interface';
+
+let lastServiceName: string = '';
+
+function ServiceProvider({
+    cloudProvider,
```

Comment:
this var name does not match the type CategoryOclVo. Please use anything else more related to the type

Reply:
fixed.